radicale_imap/__init__.py

```diff
@@ -32,16 +32,44 @@ class Auth(BaseAuth):
     type = radicale_imap
     imap_host = imap.server.tld
     imap_secure = True
+    imaps = True # for "old"-style imap over tls
 
     """
 
+    def get_connection(self, host, port):
+        return imaplib.IMAP4(host=host, port=port)
+
+    def get_secure_connection(self, host, port, imaps):
+        try:
+            if imaps:
+                connection = imaplib.IMAP4_SSL(host=host, port=port)
+            else:
+                connection = self.get_connection(host, port)
```

Comment:
All of the above must not be inside of the try-except-block. If ``secure`` is false exceptions are ignored.